src/plugins/score-plugin-gfx/Gfx/Graph/Utils.cpp

```diff
@@ -115,8 +122,8 @@ Pipeline buildPipeline(
 
   ps->setSampleCount(1);
 
-  ps->setDepthTest(false);
-  ps->setDepthWrite(false);
+  ps->setDepthTest(true);
```

Comment:
for now leave those as false - if a node needs depth test & depth write, it should enable it explicitely on its side